base/tps/src/main/java/org/dogtagpki/server/tps/processor/TPSProcessor.java

```diff
@@ -1542,8 +1541,7 @@ protected void fillTokenRecordDefaultPolicy(TokenRecord tokenRecord) throws TPSE
 
         try {
             org.dogtagpki.server.tps.TPSEngine engine = org.dogtagpki.server.tps.TPSEngine.getInstance();
-            TPSSubsystem subsystem = (TPSSubsystem) engine.getSubsystem(TPSSubsystem.ID);
-            IConfigStore configStore = subsystem.getConfigStore();
+            EngineConfig configStore = engine.getConfig();
```

Comment:
I'd suggest to use a more specific type (i.e. `TPSEngineConfig`) instead of the generic one (i.e. `EngineConfig`) whenever possible. In the past all configuration objects had `IConfigStore` type so it could get really confusing. We're slowly converting them into more specific types.

Reply:
OK, it doesn't look like that will break anything. Using the specific type gives you a specific sub-store, but you get `super(everythingElse)`

Reply:
@jmagne Am I right, or talking nonsense? :point_up:

Reply:
Spoke with Jack over IRC and he's fine with this, so merging.